### PR TITLE
feat: disable metrics

### LINKED ISSUE #1216: Add functionality to disable metrics
Let's allow users to disable/enable metrics types (jvm, storage, slice). 

Also, let's remove `route` and `key` tags from all the measurements.

### PR DESCRIPTION
**Pull request by olenagerasimova**
closes #1216

Added functionality allowing to disable some metrics types.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@dgarus please, review

File: src/main/java/com/artipie/settings/repo/RepositoriesFromStorage.java [hunk 1]
```diff
@@ -104,7 +104,7 @@ public CompletionStage<RepoConfig> config(final String name) {
                 content.key,
                 Yaml.createYamlInput(data).readYamlMapping(),
                 this.cache,
-                new MetricsContext(this.settings.meta()).enabled()
+                new MetricsContext(this.settings.meta()).storage()
```
Comment (dgarus) [inline src/main/java/com/artipie/settings/repo/RepositoriesFromStorage.java:107]: This class has to get `MetricsContext` from outside.
`MetricsContext` should be created only ones.

File: src/main/java/com/artipie/settings/MetricsContext.java [hunk 2]
```diff
@@ -53,7 +59,7 @@ public boolean enabled() {
      */
     public Optional<Pair<String, Integer>> endpointAndPort() {
         Optional<Pair<String, Integer>> res = Optional.empty();
-        final YamlMapping metrics = this.meta.yamlMapping("metrics");
+        final YamlMapping metrics = this.meta.yamlMapping(MetricsContext.METRICS);
```
Comment (dgarus) [inline src/main/java/com/artipie/settings/MetricsContext.java:62]: We should parse `yaml` only one time. 
Currently, we have to parse the same `yaml` many times when every method is called.

**Review by dgarus — CHANGES_REQUESTED** [review]
@olenagerasimova thanks!
Please look at my comments.

**Comment by olenagerasimova** [comment]
@dgarus corrected, take a look please

**Review by dgarus — APPROVED** [review]
great!

**Pull request merged**